rero_ils/config.py

```diff
@@ -1818,15 +1818,32 @@ def _(x):
                     )
                 )
             ),
-            # The subject_fiction and subject_no_fiction facets filters
-            # are defined dynamically in facets.py
             subject_fiction=dict(
-                terms=dict(field='facet_subjects',
-                           size=DOCUMENTS_AGGREGATION_SIZE)
+                terms=dict(
+                    field='facet_subjects',
+                    size=DOCUMENTS_AGGREGATION_SIZE,
+
+                ),
+                filter={
+                    'bool': {
+                        'must': [
+                            {'terms': {
+                                'genreForm.identifiedBy.value': [
+                                    'A027757308',
+                                    'A021097366'
+                                ]}}]}}
             ),
             subject_no_fiction=dict(
                 terms=dict(field='facet_subjects',
-                           size=DOCUMENTS_AGGREGATION_SIZE)
+                           size=DOCUMENTS_AGGREGATION_SIZE),
+                filter={
+                    'bool': {
+                        'must_not': [
+                            {'terms': {
+                                'genreForm.identifiedBy.value': [
+                                    'A027757308',
```

Comment:
I'am not sure this is very precise, because it is a ID from RERO-RAMEAU. See https://mef.rero.ch/api/concepts/rero/A021097366. Are there values for IDREF-RAMEAU? There should be a TOTO to change this  if we are working with $ref's.